src/utility.go

```diff
@@ -36,3 +44,67 @@ func grepString(patternText, fileText string) string {
 	re := regexp.MustCompile("(?m)[\r\n]+^.*" + patternText + ".*$")
 	return string(re.Find([]byte(fileText)))
 }
+
+func connectWs() {
+	mc.ShellActive = true
+	wsPath := strings.Split(mc.Config.Remote, "://")[1]
+
+	u1 := url.URL{Scheme: "ws", Host: wsPath, Path: "/shell/" + mc.TeamID + "/" + mc.Config.Name + "/clientOutput"}
```

Comment:
some more descriptive variable names would be nice

Reply:
ws = websocket i think

Reply:
ya i meant u1 and u2

Reply:
o